README.md

```diff
@@ -237,3 +241,30 @@ If you do not have those please consider getting a [docking station](https://eu.
    Finished successfully!
    You may find your unit on the cloud here: https://aoscloud.io/oem/units/33520
    ```
+
+### Troubleshooting
+
+#### Checking internet connection
+
+AosEdge image requires internet connection with the cloud for normal operation. To ensure that internet connection
+available on the device, login into `DomD` using serial console as described in
+[previous chapter](#provision-device-with-aoscloud). Ping `google.com`:
+
+```console
+ping google.com
+```
+
+#### Remote access
+
+AosEdge `DomD` has SSH server enabled. Once, `DomD` IP is determined, it can be accessible using SSH client. See
+[Connect to an SSH server](https://www.raspberrypi.com/documentation/computers/remote-access.html#connect-to-an-ssh-server)
+document for details.
+
+#### Retrieving system logs
+
+You can review AosEdge and system logs using the systemd journal. Run the following command to investigate potential
+issues:
+
+```console
+journalctl
+```
```

Comment:
I do believe we need to move this to a separate md file in docs and just refer it here

Reply:
I left info about internet connection and how to check it. It is important. Rest, moved to separate document.